engine/src/cmd/unit_generic.h

```diff
@@ -109,114 +113,19 @@ class AsteroidGeneric;
  * Needed by star system to determine whether current unit
  * is orbitable
  */
-enum clsptr
+enum _UnitType
```

Comment:
This would be better as `UnitType`; any particular reason for the `_`?

Reply:
This value was used in the template when I started refactoring. It was always the plan to drop it later. Opened #359 for it.